### ISSUE TITLE
adding bucket.public_url() to the wing console

### ISSUE THREAD

**Issue by tsuf239** [winglang/wing#1832]
### Community Note

> Please vote by adding a 👍 reaction to the issue to help us prioritize.
> If you are interested to work on this issue, please leave a comment.

### Feature Spec

We just added this method recently (https://github.com/winglang/wing/pull/1750) and it can be good to have it on the console as well

### Use Cases

see above

### Component

Wing Console

**Comment by skorfmann** [comment]
Yes, this would certainly be nice to have. This is just raising an error at the moment:

```
<pre>Error: Cannot provide public url for a non-public bucket<br> &nbsp; &nbsp;at ZW.publicUrl 
```

**Comment by tsuf239** [comment]
@polamoros @skyrpex Do we have it already by any chance?
